core/src/Pos/Core/Util/LogSafe.hs

```diff
@@ -71,268 +76,18 @@ module Pos.Core.Util.LogSafe
 -- what's going on.
 import           Universum
 
-import           Control.Monad.Trans (MonadTrans)
-import           Data.Foldable (Foldable, length, null)
-import           Data.List (isSuffixOf)
-import           Data.Reflection (Reifies (..), reify)
-import           Data.Text.Lazy.Builder (Builder)
-import           Formatting (bprint, build, fconst, later, mapf, (%))
+-- import           Formatting (bprint, build, fconst, later, mapf, (%))
 import qualified Formatting.Buildable
-import           Formatting.Internal (Format (..))
-import qualified Language.Haskell.TH as TH
-import           System.Wlog (CanLog (..), HasLoggerName (..), Severity (..),
-                     logMCond)
-import           System.Wlog.LogHandler (LogHandlerTag (HandlerFilelike))
+-- import           Formatting.Internal (Format (..))
 
-import           Pos.Core (Timestamp)
 import           Pos.Core.Common (Address, Coin)
+import           Pos.Core.Slotting (Timestamp)
 import           Pos.Core.Txp (TxId)
 import           Pos.Crypto (PassPhrase)
 
-----------------------------------------------------------------------------
--- Logging
-----------------------------------------------------------------------------
```

Comment:
I assume the code here is going to be coming from `Pos.Util.Log.LogSafe` instead?

Reply:
yes, `LogSafe` moved to `util`, but some instances depend on types from upper modules (i.e. `core`).